BootloaderCommonPkg/Library/PagingLib/PagingMap.c

```diff
@@ -122,6 +122,9 @@ MapMemoryRange (
 
     Address = PageBase;
     // Create 4KB pages and remap the CAR region into a different memory location
+    if (PageTable4K == NULL) {
+      return EFI_NOT_FOUND;
+    }

```

Comment:
The issues was caused by 2 “switch (Ranges[0].PageSize) {}” in same function.
Instead of add NULL check, better to merge them into one switch?